caps.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+require('dotenv').config();
+
+const PORT = process.env.PORT || 3000;
+
+const caps = require('socket.io')(PORT);
+
+// PICKUP
+caps.on('connection', (socket) => {
+  console.log(`${socket.id} Connection Established Successfully`)
+  socket.on('pickup', (payload) => {
+    console.log(`New Item Ready for Pickup \n
+    ${payload.storeName}\n 
+    ${payload.orderId}\n
+    ${payload.customerName}\n
+    ${payload.address}\n`);
+    socket.broadcast.emit('newPickup', payload);
```

Comment:
change to `caps.emit('pickup', payload);` because the socket is listening for 'pickup' to send the connection.